tools/shared.py

```diff
@@ -320,10 +320,14 @@ def fix_js_engine(old, new):
 
 # Expectations
 
-EXPECTED_LLVM_VERSION = (4, 0)
-
 actual_clang_version = None
 
+def expected_llvm_version():
+  if get_llvm_target() == WASM_TARGET:
```

Comment:
hmm, it is still ok to use the wasm backend version 4.0 (e.g., using the wasm backend in current fastcomp).

if we do this, it will warn on doing anything else but using more recent wasm backend. is that what we want? seems more common, but perhaps we should accept both? or if we're just testing 5.0, better to warn on 4.0